### PR TITLE
A few frontend improvements

### PR DESCRIPTION
**Pull request by bonomat**
1. fix: improves reference price being undefined

2. fix: moves the alert box below the grid to allow for longer messages
old: 
image

3. fix: @da-kami thought this was a cashing problem 🙈

4. fix: make the input full width
5. fix: makes the button size fix width

new

image
image

### CODE REVIEW

**Review by klochowicz — APPROVED** [review]
thanks for getting rid of these annoyances! :) 👍

File: taker-frontend/src/components/Trade.tsx [hunk 1]
```diff
@@ -214,13 +214,20 @@ export default function Trade({
                                 padding="3"
                                 spacing="6"
                             >
-                                <Button colorScheme="red" size="lg" disabled h={16}>
+                                <Button colorScheme="red" size="lg" disabled h={16} w={"40"}>
```
Comment (thomaseizinger) [inline taker-frontend/src/components/Trade.tsx:217]: Would `minWidth` maybe be a better choice to also account for large quantities?

**Review by thomaseizinger — APPROVED** [review]
Message of 2nd commit is a bit too long and thus breaks in the rendering on GitHub. If you keep the subject line to 50 chars, the commit message looks better in most places where it is being rendered.

**Review by da-kami — APPROVED** [review]

**Pull request merged**